.github/workflows/makefile.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
The name of this file `makefile.yml` does not describe its purpose

Reply:
thank you renamed the files